### ISSUE TITLE
Student Enrollment Data Not Reflected in Teacher Attendance Interface

### ISSUE THREAD

**Issue by ralhamami** [colegarien/SDDTermProject#95]
For the upcoming demo, I am assuming Nelson commented out the sorting algorithms and the hard-coded student data in my teacher attendance module, which is fine. However, in order to do the next demo, I am assuming I will need some DB data for students actually enrolled in classes, so I tested. Nelson did a decent job with connecting my module to the back end for the most part, and every step of the way went just as expected. However, students that I enrolled with were not showing up in the teacher attendance module under their enrolled course. 

Upon further investigation, I noticed that although the student enrollment interface displays "Enrolled", upon exiting and reentering, it still gives the student the option to enroll again. I would appreciate anyone's assistance in investigating this issue with me.
![enrolled](https://cloud.githubusercontent.com/assets/4790770/10747626/6396fc30-7c25-11e5-87fe-bd96b3383361.jpg)
![teacher_att](https://cloud.githubusercontent.com/assets/4790770/10747628/69db04d8-7c25-11e5-97f4-11bb4b749956.jpg)
![not enrolled](https://cloud.githubusercontent.com/assets/4790770/10747629/6f2981bc-7c25-11e5-8e91-d42bb9bf7cbd.jpg)

**Comment by symbyte** [comment]
I think it may be much too late to attempt to fix the issue now, 
especially as those who would be most instrumental to the fix may 
already be asleep etc.

This is probably something that will have to be included in the demo for 
next sprint. You are not depending on this feature from Mossi's module 
to demonstrate your progress, are you? Do you have any other way to 
demonstrate your progress?

**Comment by ralhamami** [comment]
Yeah, I think you're right. I may be able to. Let me see what I can do, and get that pull request in as soon as possible.

**Comment by nelalison** [comment]
Rayan,
           For this Sprint I was only working on student enrollments. What
you seeing in your attendance fragment is for next Sprint, was just trying
to start early on it. The attendance has a long way to go(meaning I still
have alot to implement). For your sorting I was having some bug issues with
it. I comment it out in other for you to reimplement with the new list I
will be giving to you from the cloud(sorry about that I was thinking I
mentioned it). The enrollment issue you having I will suggest you pull the
current branch from master, then try again.

**Comment by ralhamami** [comment]
Thanks Nelson, and thanks for your call. I will go ahead and run this again from the current master tonight, and I will let you know if everything goes smoothly.

**Comment by nelalison** [comment]
Issue Fixed. @ralhamami

**Comment by ralhamami** [comment]
Awesome. Good job man. :+1:
